csharp/src/Google.Protobuf.Benchmarks/WriteRawPrimitivesBenchmark.cs

```diff
@@ -150,6 +153,22 @@ public void WriteRawVarint32_WriteContext(int encodedSize)
             ctx.CheckNoSpaceLeft();
         }
 
+        [Benchmark]
+        [Arguments(1)]
+        [Arguments(2)]
+        [Arguments(3)]
+        [Arguments(4)]
+        [Arguments(5)]
+        public void WriteRawVarint32_WriteContextBufferWriter_FirstWrite(int encodedSize)
```

Comment:
Found it, looks like this is related to the WriteBufferHelper.Initialize change below. Ideally would be a separate PR, but since it's already here, let's leave it.